usaspending_api/accounts/serializers.py

```diff
@@ -115,3 +115,11 @@ class Meta:
                 "kwargs": {"read_only": True}
             }
         }
+
+
+class AgenciesFinancialBalancesSerializer(serializers.Serializer):
+
+    fiscal_year = serializers.IntegerField()
+    budget_authority_amount = serializers.DecimalField(21, 2)
```

Comment:
I've got no experience with specifying serializer columns this way, so bear with me if I'm off-base, but: is it necessary to specify the precision (especially `max_digits`) for a field in the serializer?  If it's possible, it seems more flexible to leave `max_digits` unspecified.  Or is there a benefit to specifying it I don't realize?

(It's not that I'm trying to build the API to accommodate hyperinflation...)